pootle/apps/pootle_project/models.py

```diff
@@ -339,13 +337,16 @@ def resources(self):
         '''
         cursor = connection.cursor()
         cursor.execute(sql_query, [resources_path, resources_path])
-
         results = cursor.fetchall()
 
         # Flatten tuple and sort in a list
-        resources = list(reduce(lambda x,y: x+y, results))
-        resources.sort(key=get_path_sortkey)
+        resources = set()
+        for result in results:
+            s = result[0]
```

Comment:
Nope, actually compare the whole loop to:

```python
resources = list({to_tp_relative_path(result[0]) for result in results})
```

At the same time realize that the implementation for `to_tp_relative_path()` already exists as part of the `Directory.path` property, and it is way more readable and understandable than the `s.find()...` approach proposed in this PR. Will share a follow-up commit soon.